recipes/r-taxonomycleanr/meta.yaml

```diff
@@ -0,0 +1,83 @@
+{% set version = '1.6.2' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-taxonomycleanr
+  version: {{ version|replace("-", "_") }}
+
+source:
+  git_url: https://github.com/EDIorg/taxonomyCleanr
+  sha256: a75ff50ab8ada3a0fcac7db654e0c7ec941efc061bb47c883ce66987072f0791
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ posix }}zip               # [win]
+    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
+  host:
+    - data.table >=1.12.2
```

Comment:
@Marie59 these dependencies need to be R packages. And those start usually with `r-` in conda-forge. Please check all of them if they are available.

Reply:
Thanks for your review ! I had checked and all the dependencies are available. I did the updates as you suggested !